src/ResourceManager/Profile/Commands.Profile.Test/packages.config

```diff
@@ -4,6 +4,7 @@
   <package id="Microsoft.Azure.Common" version="2.1.0" targetFramework="net45" />
   <package id="Microsoft.Azure.Common.Dependencies" version="1.0.0" targetFramework="net45" />
   <package id="Microsoft.Azure.KeyVault.Core" version="1.0.0" targetFramework="net45" />
+  <package id="Microsoft.Azure.Management.ResourceManager" version="1.6.0-preview" targetFramework="net452" />
```

Comment:
Why can't we use the default resource manager ps1 script for tests?

Reply:
removed